frontend/src/css/modules/base.scss

```diff
@@ -67,6 +67,23 @@ input.dark-autofill:-webkit-autofill:active {
     -webkit-text-fill-color: #4d4d4d !important;
 }
 
+input[type="radio"].selectTeam + label {
```

Comment:
That's because I wanted to override the materialize css styles by same property. They are not using class instead changing property directly to elements.